src/emucore/Settings.hxx

```diff
@@ -162,6 +164,9 @@ class Settings
     */
     void setSize(const string& key, const int value1, const int value2);
 
+    /** Sets all of the ALE-specific default settings */
+    void setDefaultSettings();
```

Comment:
Ok. I thought one could want to reset the system from somewhere else, but it is a silly thing. Private makes much more sense.